<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerScreen</class>
 <widget class="QMainWindow" name="ComputerScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files_rc.qrc">:/background/images/gamer_room_fase/computer_screen.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>162</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back_to_room_lb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files_rc.qrc">:/background/images/gamer_room_fase/gamer_room_img_nobg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_to_room_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="files_rc.qrc"/>
 </resources>
 <connections/>
</ui>
